jdl/jhipster/entity-options.js

```diff
@@ -28,6 +28,7 @@ const ServiceTypes = {
 };
 
 const PaginationTypes = {
+  PAGER: 'pager',
```

Comment:
```suggestion
 ```

Pager doesn't exist anymore and should be removed.

Reply:
It was always present in some templates: I just changed equality by variables :-)